io/src/main/scala/fs2/io/io.scala

```diff
@@ -131,6 +135,25 @@ package object io {
       cs: ContextShift[F]): Pipe[F, Byte, Unit] =
     writeOutputStream(F.delay(System.out), blockingExecutionContext, false)
 
+  /**
+    * Writes this stream to standard output asynchronously, converting each element to
+    * a sequence of bytes via `Show` and the given `Charset`.
+    *
+    * Each write operation is performed on the supplied execution context. Writes are
+    * blocking so the execution context should be configured appropriately.
+    */
+  def stdoutLines[F[_], O](blockingExecutionContext: ExecutionContext, charset: Charset = utf8Charset)(
+      implicit F: Sync[F],
+      cs: ContextShift[F],
+      show: Show[O]): Pipe[F, O, Unit] =
+    _.map(_.show).through(text.encode(charset)).through(stdout(blockingExecutionContext))
+
+  /** Stream of `String` read asynchronously from standard input decoded in UTF-8. */
+  def stdinUtf8[F[_]](bufSize: Int, blockingExecutionContext: ExecutionContext)(
```

Comment:
How about `stdinText` and taking a charset param defaulted to UTF8?

Reply:
Is this generalization possible ?
I though converting from bytes to string has to be done ad hoc.

Reply:
Oh, right, we don't have a `decode(c: CharSet)` method at this point in time.

Reply:
So how do I proceed ?